apps/astro-blog/src/app.css

```
@@ -24,3 +24,388 @@ html {
+.article-body h2 + p,
+.article-body h3 + p,
+.note-body h2 + p,
+.note-body h3 + p {
+  margin-top: 1rem;
+}
+
+.article-body a:not(.heading-anchor),
+.note-body a:not(.heading-anchor) {
+  color: #111;
+  text-decoration: none;
+  text-decoration-color: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)